test/masq.test.js

```diff
@@ -555,3 +574,50 @@ test('should set a watcher', async (done) => {
   masq.watch('/hello', onChange)
```

Comment:
The test misses a check on the values passed to the cb, can you add the check please ?